In [1]:
import pandas as pd

# Load the dataset
file_path = 'data.csv'
data = pd.read_csv(file_path)

# Display the first few rows of the dataset to understand its structure
data.head()


Incorporerlesplateformesdemédiassociauxetlestechnologiesémerge;Lesplateformesdemédiassociauxetlestechnologiesémergentesévoqu;Intégrerlesmédiassociauxetlestechnologiesémergentescommeoutil;Lapprentissageéducatifvialesplateformesdemédiassociauxserafaci;Lapprentissageéducatifvialestechnologiesémergentesévoquéesse;Ilmeserabénéfiquededevenirhabileàutiliserlesplateformesdeméd;Mescollèguespensentquejedevraisutiliserlesmédiassociauxetleste;Mafamilleetmesamisapprécierontmonutilisationdesmédiassociauxet;Mavieprivéeseraittransgresséesidesplateformesdemédiassociauxe;LadisponibilitédInternetetlaforcedusignalserontunproblèmelorsd;LabordabilitédesforfaitsInternetseraunproblèmedanslutilisation;Ladisponibilitéélectriqueseraunproblèmedanslutilisationdesmé;Jailescompétencestechniquespourutiliserlesplateformesdemédiass;Jeseraiprêtàutiliserlesplateformesdemédiassociauxetlestechnol;Jeseraiprêtàconsacrerletempsetlénergienécessairesàmesactivi
0                      5;5;4;4;4;5;4;5;2;5;5;4;5;4;5                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
1                      4;1;1;1;3;3;1;1;3;1;4;3;2;1;1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
2                      4;5;3;4;2;3;3;3;4;5;5;5;4;3;4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
3                      4;4;4;5;5;4;4;4;4;4;4;4;5;5;5                                                                                                                                                                                                                      

>La structure du fichier CSV semble être inhabituelle. Les données sont regroupées dans une seule colonne avec des valeurs séparées par des points-virgules. Pour une analyse appropriée, il est nécessaire de séparer ces données en colonnes distinctes, correspondant à chaque question du sondage.
>> On va procéder à cette étape de nettoyage des données pour mieux organiser l'information et faciliter l'analyse. Une fois cette étape terminée, nous pourrons examiner les résultats plus en détail.

In [2]:
# Splitting the data into separate columns
# The separator seems to be a semicolon (;)
split_data = data.iloc[:, 0].str.split(';', expand=True)

# Renaming columns based on the provided description
columns = [
    "Incorporer les plateformes de médias sociaux et les technologies émergentes améliorera ma productivité en tant qu'étudiant",
    "Les plateformes de médias sociaux et les technologies émergentes pourraient aider à faciliter l'apprentissage et à impliquer les étudiants",
    "Intégrer les médias sociaux et les technologies émergentes comme outil pédagogique augmentera ma compréhension et mon assimilation",
    "L'apprentissage éducatif via les plateformes de médias sociaux sera facile pour moi",
    "L'apprentissage éducatif via les technologies émergentes sera facile pour moi",
    "Il me sera bénéfique de devenir habile à utiliser les plateformes de médias sociaux et les technologies émergentes",
    "Mes collègues pensent que je devrais utiliser les médias sociaux et les technologies émergentes",
    "Ma famille et mes amis apprécieront mon utilisation des médias sociaux et des technologies émergentes",
    "Ma vie privée serait transgressée si des plateformes de médias sociaux et des technologies émergentes sont proposées",
    "La disponibilité d'Internet et la force du signal seront un problème",
    "L'abordabilité des forfaits Internet sera un problème",
    "La disponibilité électrique sera un problème",
    "J'ai les compétences techniques pour utiliser les plateformes de médias sociaux et les technologies émergentes",
    "Je serai prêt à utiliser les plateformes de médias sociaux et les technologies émergentes",
    "Je serai prêt à consacrer le temps et l'énergie nécessaires à mes activités d'apprentissage"
]
split_data.columns = columns

# Converting data to numeric
split_data = split_data.apply(pd.to_numeric, errors='coerce')

# Display the first few rows of the cleaned and structured dataset
split_data.head()


Incorporer les plateformes de médias sociaux et les technologies émergentes améliorera ma productivité en tant qu'étudiant  \
0                                                5.0                                                                            
1                                                4.0                                                                            
2                                                4.0                                                                            
3                                                4.0                                                                            
4                                                1.0                                                                            

   Les plateformes de médias sociaux et les technologies émergentes pourraient aider à faciliter l'apprentissage et à impliquer les étudiants  \
0                                                5.0                                                                                            
1                                                1.0                                                                                            
2                                                5.0                                                                                            
3                                                4.0                                                                                            
4                                                1.0                                                                                            

   Intégrer les médias sociaux et les technologies émergentes comme outil pédagogique augmentera ma compréhension et mon assimilation  \
0                                                4.0                                                                                    
1                                                1.0                                                                                    
2                                                3.0                                                                                    
3                                                4.0                                                                                    
4                                                3.0                                                                                    

   L'apprentissage éducatif via les plateformes de médias sociaux sera facile pour moi  \
0                                                4.0                                     
1                                                1.0                                     
2                                                4.0                                     
3                                                5.0                                     
4                                                4.0                                     

   L'apprentissage éducatif via les technologies émergentes sera facile pour moi  \
0                                                4.0                               
1                                                3.0                               
2                                                2.0                               
3                                                5.0                               
4                                                3.0                               

   Il me sera bénéfique de devenir habile à utiliser les plateformes de médias sociaux et les technologies émergentes  \
0                                                5.0                                                                    
1                                                3.0                                                                    
2                                                3.0                                                                    
3   

>Les données sont maintenant correctement organisées en colonnes distinctes pour chaque question du sondage. Ceci permet une meilleure visualisation et compréhension des réponses données par les étudiants infirmiers au Maroc sur l'utilisation des médias sociaux et des technologies émergentes dans leur apprentissage.

>>Pour fournir une interprétation approfondie, il serait utile de réaliser quelques analyses statistiques de base. Par exemple, On peux calculer la moyenne pour chaque question pour comprendre quelle est la tendance générale des réponses. De plus, On peux examiner la distribution des réponses pour voir à quel point les étudiants sont divisés ou unis dans leurs opinions.

In [3]:
# Calculating the mean for each question
mean_values = split_data.mean()

# Calculating the standard deviation for each question to understand the spread of responses
std_deviation = split_data.std()

# Creating a summary DataFrame for mean and standard deviation
summary_statistics = pd.DataFrame({
    'Mean (Average)': mean_values,
    'Standard Deviation': std_deviation
})

summary_statistics.sort_values(by='Mean (Average)', ascending=False)


Mean (Average)  \
Incorporer les plateformes de médias sociaux et...        4.232061   
Il me sera bénéfique de devenir habile à utilis...        4.186260   
Je serai prêt à consacrer le temps et l'énergie...        4.126718   
L'abordabilité des forfaits Internet sera un pr...        4.119084   
La disponibilité d'Internet et la force du sign...        4.105344   
L'apprentissage éducatif via les technologies é...        4.094656   
Intégrer les médias sociaux et les technologies...        4.079389   
Je serai prêt à utiliser les plateformes de méd...        4.065649   
Les plateformes de médias sociaux et les techno...        4.061069   
Mes collègues pensent que je devrais utiliser l...        4.045802   
J'ai les compétences techniques pour utiliser l...        3.990840   
L'apprentissage éducatif via les plateformes de...        3.977099   
Ma famille et mes amis apprécieront mon utilisa...        3.970992   
La disponibilité électrique sera un problème              3.584733   
Ma vie privée serait transgressée si des platef...        2.560305   

                                                    Standard Deviation  
Incorporer les plateformes de médias sociaux et...            0.748829  
Il me sera bénéfique de devenir habile à utilis...            0.808287  
Je serai prêt à consacrer le temps et l'énergie...            0.866010  
L'abordabilité des forfaits Internet sera un pr...            0.828313  
La disponibilité d'Internet et la force du sign...            0.860029  
L'apprentissage éducatif via les technologies é...            0.805307  
Intégrer les médias sociaux et les technologies...            0.766133  
Je serai prêt à utiliser les plateformes de méd...            0.807247  
Les plateformes de médias sociaux et les techno...            0.694625  
Mes collègues pensent que je devrais utiliser l...            0.826386  
J'ai les compétences techniques pour utiliser l...            0.891641  
L'apprentissage éducatif via les plateformes de...            0.809591  
Ma famille et mes amis apprécieront mon utilisa...            0.900602  
La disponibilité électrique sera un problème                  1.119092  
Ma vie privée serait transgressée si des platef...            1.247240

L'analyse statistique de la dataset révèle des tendances intéressantes dans les opinions des étudiants infirmiers au Maroc concernant l'utilisation des médias sociaux et des technologies émergentes dans l'éducation. Voici quelques interprétations clés basées sur les moyennes et les écarts-types calculés :

1. Haute acceptation de l'intégration technologique: Les étudiants semblent largement favorables à l'intégration des plateformes de médias sociaux et des technologies émergentes pour améliorer leur productivité en tant qu'étudiants, avec une moyenne élevée de 4.23. Cela suggère une forte reconnaissance du potentiel de ces outils dans l'amélioration de l'efficacité éducative.

2. Perception des avantages personnels et de la préparation: Les étudiants perçoivent également un bénéfice personnel à devenir compétents dans l'utilisation de ces technologies (moyenne de 4.19) et sont prêts à consacrer du temps et de l'énergie à l'apprentissage via ces plateformes (moyenne de 4.13).

3. Préoccupations liées à l'infrastructure: Les questions concernant la disponibilité d'Internet, la force du signal et l'abordabilité des forfaits Internet montrent une préoccupation notable (moyennes autour de 4.1). Cela indique que les problèmes d'infrastructure peuvent être des obstacles significatifs à l'adoption efficace de ces technologies pour l'éducation.

4. Confiance dans les compétences techniques: Les étudiants ont une confiance relativement élevée dans leurs compétences techniques pour utiliser ces technologies (moyenne de 3.99), ce qui est un bon signe pour l'adoption future de ces outils éducatifs.

5. Inquiétudes concernant la vie privée: La question sur la vie privée (moyenne de 2.56) est celle avec la moyenne la plus basse, indiquant des inquiétudes significatives sur la façon dont l'utilisation de ces technologies pourrait affecter la vie privée des étudiants.

>En résumé, cette analyse suggère que, bien que les étudiants infirmiers au Maroc soient globalement positifs quant à l'intégration des médias sociaux et des technologies émergentes dans leur éducation, il existe des préoccupations importantes concernant les infrastructures et la vie privée qui doivent être abordées pour une adoption plus large et plus efficace de ces outils.